### DISCUSSION TITLE
Can I hide the Create Copy button for the project?

### DISCUSSION THREAD

**Discussion by seleondevel** [kimai/kimai#4954] [Plugins]
![화면 캡처 2024-07-05 113623](https://github.com/kimai/kimai/assets/165639563/51b2c07a-a362-4bc9-9f5b-e2c3890fe3d3)
![스크린샷 2024-07-05 113533](https://github.com/kimai/kimai/assets/165639563/5321b57b-1972-4d41-8388-83f29ec4d2db)

Can I hide the Create Copy button for the project?

The Create copy button is suitable for backing up projects and is easy to register, but it's a feature I don't need.

I wish I could add an option to the system settings to hide the Create Copy button from the next version of Kymai, what do you think?

If you don't have a plan, how do you modify the Create Copy button to hide it?

If you accidentally click the Create Copy button, the Visible button will not work because the activity number is duplicated when copying the project.

**Comment by kevinpapst** [comment]
Almost all buttons can be customized/added/removed through events:
https://github.com/kimai/kimai/tree/main/src/EventSubscriber/Actions

Copy this class and remove this: 
https://github.com/kimai/kimai/blob/main/src/EventSubscriber/Actions/ProjectSubscriber.php#L79

Do you want to contribute to the developer documentation?
If such infos are not in there, would be great to add them
